test/widget/nav_bar/widget/nav_bar_container_test.utils.dart

```diff
@@ -0,0 +1,108 @@
+part of 'nav_bar_container_test.dart';
+
+const keyStack = Key('keyStack');
+
+final _backBtnConfig = NavBarConfig.backBtn(
+  NavBarItemBackButton(onPressed: () {}, key: const Key('back btn')),
+);
+
+final _ignoredConfig = NavBarConfig.ignored();
+
+final _singleItemConfig = createConfig();
+
+BuildContext getContext() => find.byKey(keyStack).evaluate().first;
+
+NavBarContainerState getState() =>
+    (find.byType(NavBarContainer).evaluate().first as StatefulElement).state
+        as NavBarContainerState;
+
+NavBarState getStateNavBar() =>
+    (find.byType(NavBar).evaluate().first as StatefulElement).state
+        as NavBarState;
+
+NavBarConfig createConfig({int numberOfItems = 1}) {
+  assert(numberOfItems >= 1, 'There should be at least 1 item');
+  final items = <NavBarItem>[];
+
+  for (int i = 0; i < numberOfItems; i++) {
+    final item = NavBarItemIconButton(
+      svgIconPath: 'packages/xayn_design/assets/icons/arrow_left.svg',
```

Comment:
[Looks like it's used in more than one test.](https://prnt.sc/22t1ked) It could be extracted into a constant under `test`, wdyt?